src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -436,6 +437,16 @@ private async Task StopAsyncCore(bool disposing)
             }
         }
 
+#if NETCOREAPP3_0
+        public async IAsyncEnumerable<object> StreamAsyncCore(string methodName, Type returnType, object[] args, CancellationToken cancellationToken = default)
```

Comment:
Make this method generic and have it return `IAsyncEnumerable<TResult>` so you don't need to convert later in HubConnectionExtensions.StreamAsyncCore().